pkg/messages/baseMessage.go

```diff
@@ -52,23 +57,26 @@ type StateMessage interface {
 type RequestMessage interface {
 	Message
 	Request() int
-	Reply(senderFloor int, response bool) ResponseMessage
+	Reply(senderFloor int, response bool, returnId uuid.UUID) ResponseMessage
```

Comment:
```suggestion
	Reply(senderFloor int, response bool) ResponseMessage
```

Reply:
You don't need to pass the UUID in the reply function here. Also, doing so would be wrong anyway.